apps/frontend/src/pages/people/index.tsx

```diff
@@ -209,8 +216,18 @@ const PeoplePage: React.FC = () => {
       seoTitle="People"
       header={
         <div className="flex flex-col">
-          <span className="text-label-primary text-xl leading-none font-bold tracking-[-0.1px] py-4">
+          <span className="text-label-primary text-xl leading-none font-bold tracking-[-0.1px] py-4 inline-flex">
             {`Connections (${Object.keys(connections)?.length})`}
+            <div className="justify-start items-center gap-[40-px] flex">
```

Comment:
Current: 
![image](https://github.com/user-attachments/assets/daf6fdd5-fd3e-44da-86a0-b579f37d6ead)

Designs: 
![image](https://github.com/user-attachments/assets/61e22011-471d-4d1c-9bb9-f010bfdd911b)

The export button works but I haven't figured out the styling to right-align the button.

Reply:
![image](https://github.com/user-attachments/assets/ae5b447d-4bec-4f6c-afdf-54eb97d2e3b1)